src/els_db.erl

```diff
@@ -63,7 +63,7 @@ list(Table) ->
 
 -spec store(table(), key(), any()) -> ok.
 store(Table, Key, Value) ->
-  true = ets:insert(Table, {Key, Value}),
+  ets:insert(Table, {Key, Value}),
```

Comment:
Was this  pattern matching removed because with a `bag` table it's no longer always `true`?

Reply:
That is correct.